src/App.tsx

```diff
@@ -26,32 +60,34 @@ function App() {
         type: 'ACCEPT_COOKIES'
       });
     }
+
     const getAirtableRecords = async () => {
-      const response = await api.getAirtableRecords()
-      dispatch({
-        type: 'GET_AIRTABLE_RECORDS',
-        payload: response
-      });
+      if (dataPageState.routingOccurred) {
```

Comment:
Document this weirdness somewhere so we can come up with ideas on how to make this better